frontend/pages/admin/OrgSettingsPage/cards/Statistics/Statistics.tsx

```diff
@@ -39,6 +39,10 @@ const Statistics = ({
         live_query_disabled:
           appConfig.server_settings.live_query_disabled || false,
         enable_analytics: enableUsageStatistics,
+        deferred_save_host: appConfig.server_settings.deferred_save_host,
+        query_reports_disabled:
+          appConfig.server_settings.query_reports_disabled,
+        scripts_disabled: appConfig.server_settings.scripts_disabled,
```

Comment:
consider checking if this is a found bug

Reply:
that bug would be that these fields are required but weren't being sent before?